v2/printings/msbp.json

```diff
@@ -28,21 +25,6 @@
   {
     "card_id": "deep_dive",
     "card_set_id": "msbp",
-    "cost": 2,
-    "deck_limit": 3,
-    "id": "32003",
-    "illustrator": "Cat Shen",
-    "position": 3,
```

Comment:
No position 3 card in this set?

Reply:
?  I don't understand.

Reply:
The card above is position 2, the card below is position 4. What card will be in the position 3/120?